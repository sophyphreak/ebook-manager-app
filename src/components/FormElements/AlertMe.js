import React, { Component } from 'react';
import CheckboxInput from './MainElements/CheckboxInput';

const AlertMe = (props) => (
  <CheckboxInput
    {...props}
  />
);

export default AlertMe;