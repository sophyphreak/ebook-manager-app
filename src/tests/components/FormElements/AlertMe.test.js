import React from 'react';
import { shallow } from 'enzyme';
import AlertMe from '../../../components/FormElements/AlertMe';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<AlertMe />);
});

test('should render AlertMe correctly', () => {
  expect(wrapper).toMatchSnapshot();
});