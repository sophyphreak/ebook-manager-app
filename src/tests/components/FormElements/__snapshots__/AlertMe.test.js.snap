// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AlertMe correctly 1`] = `<CheckboxInput />`;
